python/oneflow/framework/docstr/math_ops.py

```diff
@@ -1240,3 +1240,33 @@
         
     """,
 )
+
+
+add_docstr(
+    oneflow.dot,
+    r"""This operator computes the dot product of tensor input and other.
+    
+    The equation is:
+    
+	$$		
+	​   \\sum_{i=1}^{n}(x[i] * y[i])
```

Comment:
```suggestion
	​   \sum_{i=1}^{n}(x[i] * y[i])
```

看是不是改成单斜杠才是最优的（make html 测试下看）。因为这里是 r 字符串，不需要转义。

Reply:
需要转义